src/WebJobs.Extensions.DurableTask/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    // TODO: Use IConfigurationExtensions in WebJobs SDK when exposed
```

Comment:
Is WebJobs.Host v3.0.32 not available yet? This method was added in that version so we might be able to just update the version in the extension (not sure what else breaks with this change though).

Reply:
Unfortunately I don't see this method in the latest nuget package for some reason. I decompiled 3.0.30 and do not see the method `GetWebJobsConnectionSection`, but I see it in the [dev branch](https://github.com/Azure/azure-webjobs-sdk/blob/bba8d6a69f77f3fea870082efcd540a138a788b4/src/Microsoft.Azure.WebJobs.Host/Extensions/IConfigurationExtensions.cs#L103).
![image](https://user-images.githubusercontent.com/2955179/143935221-be867f06-0c64-4880-b440-bf3c2a7c2734.png)

Reply:
Could we try adding WebJobs.Host v3.0.31? That should have the GetWebJobsConnectionSection method and it should be publicly available https://www.nuget.org/packages/Microsoft.Azure.WebJobs/

Reply:
Ah yes! That worked just fine